Detectors/GlobalTrackingWorkflow/tpcinterpolationworkflow/src/TPCInterpolationSpec.cxx

```diff
@@ -0,0 +1,259 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// @file  TPCInterpolationSpec.cxx
+
+#include <vector>
+
+#include "Framework/ControlService.h"
+#include "DataFormatsITS/TrackITS.h"
+#include "ReconstructionDataFormats/TrackTPCITS.h"
+#include "DataFormatsTPC/TrackTPC.h"
+#include "DataFormatsTPC/ClusterNative.h"
+#include "DataFormatsTPC/ClusterNativeHelper.h"
+#include "DataFormatsTPC/TPCSectorHeader.h"
+#include "DetectorsBase/GeometryManager.h"
+#include "DetectorsBase/Propagator.h"
+#include "TPCInterpolationWorkflow/TPCInterpolationSpec.h"
+
+using namespace o2::framework;
+
+namespace o2
+{
+namespace tpc
+{
+
+void TPCInterpolationDPL::init(InitContext& ic)
+{
+  //-------- init geometry and field --------//
+  o2::base::GeometryManager::loadGeometry();
+  o2::base::Propagator::initFieldFromGRP("o2sim_grp.root");
+
+  mInterpolation.setDPLIO(true);
```

Comment:
just for your info: in the ITS/TPC matching I've left only the mode of running with externally provided input data, i.e. "DPLIO", suppressing the part with "standalone" running from the macro, as supporting the optional internal reading from the tree adds lot of headache and messes up the code. 

See changes of MatchTPCITS.h/cxx in https://github.com/AliceO2Group/AliceO2/commit/e0300e21a26315e60bf09b1f94b17ecb33a17929#diff-a723c1b696298055e4e2e74045c5a499

Reply:
I did the same now for the `TrackInterpolation`. It makes the code much clearer I think.